readme.md

```diff
@@ -6,6 +6,10 @@
 1. Download and open this [example project](https://github.com/samsface/godot-steam-api/archive/refs/heads/master.zip). Or download through Godot using the [Asset Library](https://godotengine.org/asset-library/asset/1020).
 3. There will be a SteamAPI tab in the Project Settings, follow the instructions to configure the SteamAPI for your game.
 
+The path for the SDK needs to be the root directory of the Steam SDK (the folder containing the README file).
+
+Note that if you're on Windows, the path needs to be formatted in the Godot/Linux style, e.g. if your SDK is located at `c:\temp\steamworks_sdk_152`, you want to specify the path as `/c/temp/steamworks_sdk_152`.
```

Comment:
@nightblade9 Thanks for the pr, will include the tip to make it more clear which directory is needed.

But on this path format, are you sure about this? I've talked to two people who used this on windows and they've never had this issue. Is it possible the lower case 'c' in your path caused the issue?
![image](https://user-images.githubusercontent.com/4586101/140830957-be556bfc-b672-42af-b7b6-5e9426bcd3b1.png)

Reply:
> are you sure about this? I've talked to two people who used this on windows

I'm on Windows too. No, I'm not sure about this. I deleted the plugin, all the settings, rolled back my code, etc. and reinstalled everything clean (assetlib, activate plugin, configure). No combination of paths (Godot style or Windows style, upper-case or lower-case C) seems to work. I have Steam running. I'll investigate further later.

I'm no longer sure if it's the path style that's the problem, although I think the exact directory you should specify could be clearer.